tokens/nft-minter/anchor/tests/test.ts

```diff
@@ -25,7 +25,7 @@ describe('NFT Minter', () => {
 
     const transactionSignature = await program.methods
       .mintNft(metadata.name, metadata.symbol, metadata.uri)
-      .accounts({
+      .accountsPartial({
```

Comment:
Is this because of anchor 0.30.x's pda auto resolution?

Reply:
Yes. Added just to be explicit, in the example, on the account to pass in the instruction